apps/emqx_connector/src/emqx_connector_resource.erl

```diff
@@ -168,6 +169,9 @@ start(Type, Name) ->
 %     create(ConnectorType, ConnectorName, Conf).
 
 create(Type, Name, Conf) ->
+    create(Type, Name, Conf, #{}).
+
+create(Type, Name, Conf, Opts) ->
```

Comment:
What is Opts for?

Reply:
It's used by both `update/4` and then `recreate/4` so I put it there rather than removing it from the others.